dopamine/experiment4/experiment4b/experiment4b_analysis.jl

```diff
@@ -25,23 +25,29 @@ files.id = [
 
 
 # DEFINE EXPERIMENT DIRECTORY
-experimentdir = "./22spring/experiment4/experiment4b/"
+experimentdir = @__DIR__
 
 
 
-# COMPILE DATAFRAME WITH 5 POSITION MEASUREMENTS EVERY SEC
-data = DataFrame(id=String[], track=Int[], xpos=Float64[], ypos=Float64[])
+# CREATE DATAFRAME FROM CSV
+alldata = DataFrame(id=String[], track=Int[], xpos=Float64[], ypos=Float64[])
 
-positionsdir = joinpath(experimentdir, "data/Position")
+positionsdir = joinpath(experimentdir, "data", "Position")
 
 for row in eachrow(files)
-    load_tracks!(data, joinpath(positionsdir, string(row.num, "Position.csv")), row.id)
+    load_tracks!(alldata, joinpath(positionsdir, string(row.num, "Position.csv")), row.id)
 end
 
+# TAKE EVERY 5 POSITION MEASUREMENTS
+data = alldata[5:5:end, :]
```

Comment:
This is every 5 positions across every track, right? It think you want to do within tracks...